### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by TheGreenBandit**
removed unused things, also removed yimmenu readme part as its not needed

### CODE REVIEW

**Comment by tupoy-ya** [comment]
I think we need to finish https://github.com/YimMenu/YimMenu/pull/539 first.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,10 @@
 # TupoyeMenu made possible by [YimMenu](https://github.com/YimMenu/YimMenu)
 ![](https://img.shields.io/badge/dynamic/json?color=ffab00&label=Online%20Version&query=%24.game.online&url=https%3A%2F%2Fraw.githubusercontent.com%2FYimMenu%2FYimMenu%2Fmaster%2Fmetadata.json&style=flat-square&labelColor=000000) ![](https://img.shields.io/badge/dynamic/json?color=ffab00&label=Game%20Build&query=%24.game.build&url=https%3A%2F%2Fraw.githubusercontent.com%2FYimMenu%2FYimMenu%2Fmaster%2Fmetadata.json&style=flat-square&labelColor=000000)
 
-A garbage fork of YimMenu made by an idiot
+A garbage fork of YimMenu made by an idiot (genious)
```
Comment (tupoy-ya) [inline README.md:4]: What?

Comment (TheGreenBandit) [reply README.md:4]: You want it removed? Thought it was funny

Comment (tupoy-ya) [reply README.md:4]: I'm a skid not genious

Comment (TheGreenBandit) [reply README.md:4]: Same except im even more dumb

**Pull request closed without merging**